### DISCUSSION TITLE
Fetching results from specific sources ONLY

### DISCUSSION THREAD

**Discussion by arda618** [searx/searx#3503] [Development]
I want to modify the source code of my SearXNG or SearX instance to get results only from the websites of my wish (what I mean is NOT an engine, but a website that contains searchable data). Let's say I only have Google enabled and I only want a list of websites to be indexed, and ignore all the other results that are not from the websites that I've listed. What do I have to do to achieve that?

**Comment by unixfox** [comment]
Issue created in https://github.com/searx/searx/issues/3502
